src/btree/bt_vrfy.c

```diff
@@ -42,13 +42,15 @@ typedef struct {
     int verify_err;
 } WT_VSTUFF;
 
+static int __verify_compare_page_id(const void *, const void *);
```

Comment:
Can you please sort these functions? `__verify_page_content_int` is not at the right position either.

Reply:
Sorted them in [ca05176](https://github.com/wiredtiger/wiredtiger/pull/12419/commits/ca0517656f8d4faa28f89f578fb4c7108477f156) :)